### ISSUE TITLE
problems on finetuning on custom data

### ISSUE THREAD

**Issue by xxAna** [eriklindernoren/PyTorch-YOLOv3#459]
Thank you for your sharing! It was a wonderful job! 
I have a problems on finetuning on custom data. If I use the pretrained weights to init and train on my custom data, after training , can the model still have the ability to detect the classes in COCO?And if several custom classes are overlap with COCO classes, how can I keep their class id ? For example, if the id of class "people" is 70 in coco dataset, while my custom data only have 40 classes and the class "people" is also included, how to set its id?Should it also set to be 70? While I didn't find a file to set the id and seems they just default to begin from the beginning. That is, in classes.names, from the 1st
row, the id is 0.
Waiting for your reply!Best wishes!

**Comment by ly0303521** [comment]
> Thank you for your sharing! It was a wonderful job!
> I have a problems on finetuning on custom data. If I use the pretrained weights to init and train on my custom data, after training , can the model still have the ability to detect the classes in COCO?And if several custom classes are overlap with COCO classes, how can I keep their class id ? For example, if the id of class "people" is 70 in coco dataset, while my custom data only have 40 classes and the class "people" is also included, how to set its id?Should it also set to be 70? While I didn't find a file to set the id and seems they just default to begin from the beginning. That is, in classes.names, from the 1st
> row, the id is 0.
> Waiting for your reply!Best wishes!

classes.names is used to set the ID, like below,  people ID is 0, dog ID is 1, cat ID is 2
people
dog
cat

**Comment by ly0303521** [comment]
have you used detect.py, my cls_conf is always 1.0, do you know why
+ Label: without_mask, Conf: 1.00000
